graylog2-web-interface/src/components/field-analyzers/FieldQuickValues.jsx

```diff
@@ -14,7 +14,15 @@ import RefreshStore from 'stores/tools/RefreshStore';
 
 const FieldQuickValues = React.createClass({
   propTypes: {
-    permissions: PropTypes.arrayOf(React.PropTypes.string),
+    from: PropTypes.any.isRequired,
+    to: PropTypes.any.isRequired,
+    resolution: PropTypes.any.isRequired,
+    stream: PropTypes.object,
+    permissions: PropTypes.arrayOf(PropTypes.string).isRequired,
+    query: PropTypes.string.isRequired,
+    page: PropTypes.number.isRequired,
+    rangeType: PropTypes.string.isRequired,
+    rangeParams: PropTypes.object.isRequired,
```

Comment:
I'd vote for only declare in the component's `propTypes` the props that are actually used, as it makes more clear what is actually used by the plugin. In my opinion we should leave the declaration of what is passed to each kind of plugin to the documentation that we need to write.